tools/cldr/lib/install-cldr-jars.sh

```diff
@@ -67,9 +66,10 @@ which mvn > /dev/null || die "Cannot find Maven executable 'mvn' in the current
 (( $# == 1 )) && [[ -d "$1" ]] || die "Usage: ./install-cldr-jars.sh <CLDR-root-directory>"
 
 # Set up a log file (and be nice about tidying it up).
-LOG_FILE="$(tempfile)" || die "Cannot create temporary file!"
-trap "rm -f -- '${LOG_FILE}'" EXIT
```

Comment:
I got rid of the tidyup since now the log file isn't a random name, it seems fine to leave it in /tmp and avoid "polluting" the lib/ directory with a "last_log" file. I'm easy either way so if anyone prefers the copy and tidyup behaviour, just let me know.